### ISSUE TITLE
Cordova will not play video on iOS.

### ISSUE THREAD

**Issue by Tarang** [meteor/meteor#5143]
I've tested the equivalent on a vanilla cordova app with the following html added along with BigBuckBunny_320x180.mp4 from http://download.blender.org/peach/bigbuckbunny_movies/

This works without issue on cordova 
```
 <video playsinline webkit-playsinline preload src="BigBuckBunny_320x180.mp4"
             autoplay
             loop
             height="270" width="375">
</video>
```

The equivalent on a vanilla meteor (1.1.0.3 or 1.2-rc14) project wont work, when the file is stored in `/public`.

I suspect this has something to do with how `file://` is handled.

To reproduce simply create vanilla projects of either, download file `BigBuckBunny_320x180.mp4` and place it in `/public` (meteor) and `/` (cordova), add the iOS platforms to both and run them.

**Comment by martijnwalraven** [comment]
This is a known issue, and something I hope to fix after the Meteor 1.2 release as part of Hot Code Push improvements for Cordova. Also see #3799.

**Comment by Tarang** [comment]
@martijnwalraven I hope this also fixes for `assets-library://` urls too.
